packages/gatsby-cli/src/create-cli.js

```diff
@@ -247,6 +247,12 @@ function buildLocalCommands(cli, isLocalSite) {
     },
   })
 
+  cli.command({
+    command: `clean`,
+    desc: `Wipe the local gatsby environment when something has gone wrong`,
```

Comment:
```suggestion
    desc: `Wipe the local gatsby environment including built assets and cache`,
```

Probably makes sense to make this description independent of the typical use case?